Sp docs improvements (#114)
checkly/docs: communicate/status-pages/customization.mdx

sidebarTitle: 'Customization'
---

## Custom domain

<Note>
Custom domains are available on Communicate Starter and higher plans. [View pricing](https://checklyhq.com/pricing)
</Note>

You can host your Status Page under your own domain. To make this work, you need to do a few things:

<Steps>
<Step title="Add a valid custom domain to your Status Page">
![Custom domain configuration](/images/docs/images/status-pages/status-pages-custom-domain-1.png)
</Step>

<Step title="Create a CNAME record in your DNS that points to `custom-domain.checkly-status-page.com`">
Any DNS provider will have the option to easily add CNAME records. For example on Cloudflare this looks as follows.
![Custom domain CNAME configuration](/images/docs/images/status-pages/status-pages-custom-domain-2.png)
If your provider is Cloudflare, you also need to disable the proxy on the CNAME, it should always be set to DNS only.
</Step>

<Step title="Verify you are the owner of the domain by adding a first `TXT` record to your DNS">

> The owner verification TXT record details can be copied from the Status Page configuration section page after providing a custom domain and saving the page once.

Reviewer: ```suggestion
Copy the domain-verification TXT record from the Status Page configuration after you add your custom domain and save the page.
```

<Note>
Your custom domain needs this verification in order for search engines to list your Status Page.
</Note>
</Step>
<Step title="Add validation `TXT` record(s)">
The validation TXT record details can be copied from the Status Page configuration section after providing a custom domain and saving the page once.
![Custom domain TXT configuration](/images/docs/images/status-pages/status-pages-custom-domain-3.png)
</Step>
</Steps>

## Customize your Status Page to match your brand identity

- Upload your company logo and the link it should direct to
- Upload a custom favicon

> - Choose the default theme: auto (system), light or dark theme

Reviewer: ```suggestion
- Choose the default theme: auto (system), light or dark
```
